nmtwizard/preprocess/tokenizer.py

```diff
@@ -63,3 +63,13 @@ def make_subword_learner(subword_config, subword_dir, tokenizer=None):
         "subword_type": subword_type,
         "size": vocab_size
     }
+
+def load_vocabulary(state, vocab_name):
```

Comment:
The interface should probably be `load_vocabulary(vocabulary_path)`. `state` is very specific to the pipeline object.